docs/CHANGELOG.md

```diff
@@ -5,6 +5,88 @@ All notable changes and improvements to the Job Tracker Frontend project are doc
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.187.0] - 2025-08-02
```

Comment:
Just to make the same version number as the PR number, change the [0.187.0] to [0.188.0]